### PR TITLE
Typos and wording

### PR DESCRIPTION
**Pull request by linustata**
Some small changes, fixing typos and updating some wordings to flow a bit more smoothly.

In episode 4 there are a couple of mentions of repositories, before any explanation of what they are. I couldn't think of a good way to rephrase them, it's not a big problem but any suggestions welcome! (There's a quick explanation later in the episode under "Default Git branch naming", but I think moving the explanation higher up may feel a bit awkward.)
```
This user name and email will be associated with your subsequent Git activity,
which means that any changes pushed to GitHub (or another Git host server)
from here on will include this information. If you work on public repositories
with these services, then this information will be publicly visible.
```

```
The commands we just ran above only need to be run once: the flag `--global` tells Git
to use the settings for _every_ Git repository, in your user account, on this computer.
```

### CODE REVIEW

File: _episodes/05_making_repos.md [hunk 1]
```diff
@@ -34,11 +34,11 @@ It is perfectly possible to create a Git repository on your own computer and hav
...
 that you develop within the repository, but doesn't provide a way to easily
 share this developmental history with others, or collaborate with them on code.
-By contrast, the most common way that a person uses Git is to have a local repository
-that is linked to a remote repository. A typical example here is
+The most common way that a person uses Git is instead to have a local repository
+that is linked to a remote repository. A typical example is
 where there is a remote repository on GitHub and you have a local version of the
 repository on your own computer. You keep the remote repository up-to-date by
-'pushing' updates you make to your local repository to it (we will look more at
+'pushing' updates made on your local repository (we will look more at
```
Comment (stephenpcook) [inline _episodes/05_making_repos.md]: nit: maybe `'pushing' updates made on your local repository to the remote repository` adds some clarity?

**Review by stephenpcook — APPROVED** [review]
Nice clarifications. Tiny nit, otherwise LGTM

File: _episodes/12_ignoring_files.md [hunk 2]
```diff
@@ -259,7 +259,7 @@ Your branch is ahead of 'origin/main' by 2 commits.
 nothing to commit, working tree clean
 ```
 
-Notice how we still have 2 some commits that have not been pushed to the remote
+Notice how we still have some commits that have not been pushed to the remote
 repository. We'll tidy up by pushing these commits, so that our local repository
 and remote repository are up-to-date with each other:
```
Comment (42john42) [inline _episodes/12_ignoring_files.md:265]: Nit: This really is a nit, but wouldn't you need a push _and a pull_ to really make sure that the local and remote are up to date _with each other_? 
Also, not sure that _tidying up_ really describes this part of the process. I think it's more fundamental than that.
Suggestion: `We'll update the remote repository now:`

**Review by 42john42 — APPROVED** [review]
Just a very picky nit, which you could take or leave.

File: _episodes/12_ignoring_files.md (hunk 2 above)
Comment (linustata) [reply _episodes/12_ignoring_files.md:265]: Very good points - changed the wording around 'tidying', left it just as pushing for the time being for simplicity, but will keep it in mind when revisiting the material in the future!

**Pull request merged**